backend/.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 
 # Ignore unused imports
-ignore = F401
+ignore = F401,W504
```

Comment:
what is this doing here please add it also to the comment

Reply:
have added comments